examples/advanced/main.tf

```diff
@@ -103,10 +130,79 @@ module "buckets" {
       kms_guid                      = module.key_protect.kms_guid
       kms_key_crn                   = module.key_protect.keys["${local.key_ring_name}.${local.key_name}"].crn
       skip_iam_authorization_policy = true # Auth policy created in first bucket
+      cbr_rules = [{
+        description      = "CBR rule for ICL metrics bucket"
+        enforcement_mode = "report"
```

Comment:
we should be able to enable this right? Since its scoped to the instance created by the example